tests/compile-and-dump/Promote/GenDefunSymbols.ghc84.template

```diff
@@ -9,7 +9,7 @@ Promote/GenDefunSymbols.hs:0:0:: Splicing declarations
         = snd ((GHC.Tuple.(,) LiftMaybeSym1KindInference) GHC.Tuple.())
     data LiftMaybeSym1 (l :: TyFun a0123456789876543210 b0123456789876543210
                              -> Type) (l :: TyFun (Maybe a0123456789876543210) (Maybe b0123456789876543210))
-      = forall arg. Data.Singletons.Internal.SameKind (Apply (LiftMaybeSym1 l) arg) (LiftMaybeSym2 l arg) =>
+      = forall arg. singletons-2.4.1:Data.Singletons.Internal.SameKind (Apply (LiftMaybeSym1 l) arg) (LiftMaybeSym2 l arg) =>
```

Comment:
Can we scrub the version number from these files before comparing? That seems more robust.

Reply:
Oh dear, I somehow missed that. Yes, we should absolutely scrub versions out using the `sed` machinery.